### PR TITLE
[sailfish-setup] Add default user to sailfish-radio group. Contribute…

### PR DESCRIPTION
**Pull request by mkenttala**
…s to JB#47593

### CODE REVIEW

File: rpm/sailfish-setup.spec [hunk 1]
```diff
@@ -24,6 +24,7 @@ if ! getent passwd sailfish-mdm >/dev/null ; then
     useradd -r -g sailfish-mdm -G privileged -d / -s /sbin/nologin sailfish-mdm || :
 fi
 groupadd -rf sailfish-radio || :
+usermod -a -G sailfish-radio nemo || :
```
Comment (monich) [inline rpm/sailfish-setup.spec]: Wouldn't that give nemo the right to call privileged functions in ofono D-Bus interface? Or is the plan to make sailfish-radio a non-privileged group? What's the purpose of having this group in the first place then?

Comment (mkenttala) [reply rpm/sailfish-setup.spec]: Are there some interfaces that shouldn't be available for nemo? This change is related to https://git.merproject.org/mer-core/commhistory-daemon/merge_requests/30/diffs.

Comment (monich) [reply rpm/sailfish-setup.spec]: Yes, things like `org.ofono.MessageManager.SendMessage`  and `org.ofono.VoiceCallManager.Dial` for example are disallowed for non-privileged processes. And `sailfish-radio` group is kind of considered privileged too, although I was against that.

Comment (mkenttala) [reply rpm/sailfish-setup.spec]: Pushed a new version where a new group is used for the MMS user service.

Comment (monich) [reply rpm/sailfish-setup.spec]: So the plan is that `sailfish-radio` group controls access to ofono, `sailfish-mms` controls access to mms-engine, and commhistoryd has **both** `sailfish-mms` and `sailfish-radio` euids at the same time, right?

File: rpm/sailfish-setup.spec [hunk 2]
```diff
@@ -25,5 +25,8 @@ if ! getent passwd sailfish-mdm >/dev/null ; then
 fi
 groupadd -rf sailfish-radio || :
 
+groupadd -f sailfish-mms || :
+usermod -a -G sailfish-mms nemo || :
```
Comment (monich) [inline rpm/sailfish-setup.spec:29]: Hmm... If `sailfish-mms` group is supposed to control access to privileged functions of mms-engine then a non-privileged user shouldn't be part of this group. If `sailfish-mms` is not supposed to control access to privileged functions of mms-engine then why is it needed in the first place. I start having a feeling that I'm missing something.

Comment (mkenttala) [reply rpm/sailfish-setup.spec:29]: My understanding is that with this new group we can control which users can use MMS. Not used at the moment  but in the future with multiuser support there can be those too.

Comment (monich) [reply rpm/sailfish-setup.spec:29]: If it's not used at the moment and there's no clear plan how it's going to be used, then let's forget it for now and get back to it when it starts making sense!

Comment (monich) [reply rpm/sailfish-setup.spec:29]: Removing things is so much harder that adding them. Once you add something, it's a matter of time when something starts to depend on it and that's how unnecessary or simply wrong things pile up and litter the platform.

**Comment by Tomin1** [comment]
@mkenttala, do we still need this?

**Comment by mkenttala** [comment]
> @mkenttala, do we still need this?

Probably not, at least things have changed so much that this is obsolete now.

**Pull request closed without merging**